tests/IntegrationTest.php

```diff
@@ -73,4 +74,53 @@ public function gettingEncryptedStuffFromGoogleShouldWork()
         $this->assertTrue($connected);
         $this->assertRegExp('#^HTTP/1\.0#', $response);
     }
+
+    /** @test */
+    public function testSelfSignedRejectsIfVerificationIsEnabled()
```

Comment:
I'd rather not just skip because it's failing. HHVM is skipped because the future is not supported in the language.